ext/mbstring/mbstring.c

```diff
@@ -3166,6 +3166,138 @@ PHP_FUNCTION(mb_rtrim)
 	php_do_mb_trim(INTERNAL_FUNCTION_PARAM_PASSTHRU, MB_RTRIM);
 }
 
+PHP_FUNCTION(mb_levenshtein)
+{
+	zend_string *string1, *string2, *enc_name = NULL;
+	zend_long cost_ins = 1;
+	zend_long cost_rep = 1;
+	zend_long cost_del = 1;
+
+	ZEND_PARSE_PARAMETERS_START(2, 6)
+		Z_PARAM_STR(string1)
+		Z_PARAM_STR(string2)
+		Z_PARAM_OPTIONAL
+		Z_PARAM_LONG(cost_ins)
+		Z_PARAM_LONG(cost_rep)
+		Z_PARAM_LONG(cost_del)
+		Z_PARAM_STR_OR_NULL(enc_name)
+	ZEND_PARSE_PARAMETERS_END();
+
+	if (ZSTR_LEN(string1) == 0) {
+		RETVAL_LONG(ZSTR_LEN(string2) * cost_ins);
```

Comment:
This is not right. `ZSTR_LEN` is the number of bytes in `string2`. You want the number of unicode codepoints instead.

Reply:
Thank you. I'll fix later.

Reply:
fixed.